services/terrain/src/terrain/clients/data_info.clj

```diff
@@ -148,6 +148,12 @@
   (log/warn "checking to see if" path "can be created")
   (st/can-create-dir? user path))
 
+(defn overwrite-file
```

Comment:
This has a version in the plain client, rather than the raw client, to convert the path to a UUID.